contrib/babelfishpg_common/sql/upgrades/babelfish_common_helper--5.1.0--5.2.0.sql

```diff
@@ -4,3 +4,13 @@
 
 -- complain if script is sourced in psql, rather than via ALTER EXTENSION
 \echo Use "ALTER EXTENSION ""babelfishpg_common"" UPDATE TO '5.2.0'" to load this file. \quit
+
+SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+-- For JSON Functions
+SET enable_domain_typmod = TRUE;
```

Comment:
Is it OK to keep this as open ?

Reply:
seems this is the norm for `CREATE DOMAIN` statements in the extension